template.php

```diff
@@ -138,7 +138,7 @@ public static function render_footer($admin = false)
     <div id="footerwrap">
       <div class="container">
         <div class="row centered">
-          <div class="col-md-4 text-left"><a href="https://github.com/Pryx/server-status/" target="_blank"><i class="fab fa-github" aria-hidden="true"></i>&nbsp;</a>Copyright © <?php echo date("Y");?> Vojtěch Sajdl</div>
+          <div class="col-md-4 text-left"><a href="https://github.com/Pryx/server-status/graphs/contributors" target="_blank">Copyright © <?php echo date("Y");?> Server Status Project Contributors </a><?php if(strlen($additional_copyright)>1){ echo " and ".$additional_copyright; } ?></div>
```

Comment:
Ternary operator and `empty()` seem like a better fit to me, but otherwise it looks great :)

Reply:
That is because if people defines it as " " (with a space) instead of "" (completely empty).